google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/ManagedCloudSdkUpdateService.java

```diff
@@ -97,6 +88,24 @@ void schedule(TimerTask timerTask, long delay, long period) {
     return new javax.swing.Timer(delayMillis, null);
   }
 
+  /**
+   * Calculates milliseconds delay before first update based on last update/install time. If
+   * complete update interval or more elapsed, first update is scheduled immediately (returns 0).
+   */
+  private long getDelayBeforeFirstUpdate() {
+    Optional<Long> lastTimeOfUpdate =
+        CloudSdkServiceUserSettings.getInstance().getLastAutomaticUpdateTimestamp();
+    long delayBeforeUpdateMillis = SDK_UPDATE_INTERVAL_MS;
+    if (lastTimeOfUpdate.isPresent()) {
```

Comment:
nit: 

```
lastTimeOfUpdate.ifPresent( .. -> ..);
```

Reply:
Actually let's leave it as is - the thing is we need to assign to outer variable or return value from this block, and we won't be able to do that from lambda, so it might be easier to read as it is now.